src/main/java/org/fundacionjala/coding/abel/movies/Customer.java

```diff
@@ -0,0 +1,98 @@
+package org.fundacionjala.coding.abel.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Unknown on 5/26/2017.
+ */
+public class Customer {
+
+    private String name;
+    private List<Rental> rentals;
+    private double totalAmount;
+    private int frequentRenterPoints;
+
+    /**
+     * This is the default constructor.
+     * @param name String for customers name.
+     */
+    public Customer(String name) {
+        this.name = name;
+        rentals = new ArrayList<>();
+    }
+
+    /**
+     * Method for add Rentals.
+     * @param rental Instance of Rental.
+     */
+    public void addRental(Rental rental) {
+        rentals.add(rental);
+    }
+
+    /**
+     * Getter for Customer Name.
+     * @return Name of Customer.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * .
+     * @return Statement.
+     */
+    public String statement() {
+        String headerStatement = String.format("Rental Record for %s%s", getName(), System.lineSeparator());
+        String bodyStatement = bodyStatement();
+        String footerStatement = String.format("Amount owed is %.1f%sYou earned %d frequent renter points",
+                totalAmount, System.lineSeparator(), frequentRenterPoints);
+        return String.join("", headerStatement, bodyStatement.toString(), footerStatement);
+    }
+
+    /**
+     * Something.
+     * @return String.
+     */
+    private String bodyStatement() {
```

Comment:
This method is still doing 3 things, it is calculating the totalAmount, totalFrequentRenterPoints and it is printing.
Please split it.